configs/classification/imagenet/resnet/resnet50_b32x8_100e_jpg.py

```diff
@@ -1,4 +1,4 @@
-_base_ = 'configs/base.py'
+_base_ = '../../../base.py'
```

Comment:
is there anything wrong with configs/base.py ?  It is the expected behaviour  when using easycv after piping install

Reply:
yes, if this file is used as the base file of other files, it will not be found due to relative path problems